third_party/boringssl/BUILD.generated.gni

```diff
@@ -267,6 +264,14 @@ crypto_sources = [
   "src/third_party/fiat/internal.h",
 ]
 
+if (is_native_target_build) {
+crypto_sources += [
```

Comment:
Let's indent this block and instead of adding the sources, remove the source when the converse of the condition is true (do crypto_sources -= [] when !is_native_target_build).

Also, did `is_starboard` not work as a conditional? I agreed that using that would probably be better

Reply:
Ok, yes subtractive does result in a more contained diff.

Didn't actually try `is_starboard`, did that now and it also does work, changed.